app/models/reducers/external_reducer.rb

```diff
@@ -21,7 +21,8 @@ class ExternalReducerFailed < StandardError; end
       end
     end
 
-    def reduction_data_for(extractions)
+    def reduction_data_for(extractions, reduction)
+      #TODO: is this the right idea here?
```

Comment:
I'm okay with this, but suggest adding a validation to ensure nobody ever sets it to be a running_reducer until it actually supports it.

Reply:
I actually wanted to ask about this--should we expect someone who is implementing an external reducer to implement their own store? It seems redundant for caesar to maintain one.

Reply:
Maybe. But we can leave that for another PR where we tackle adding running reductions for external reducers.

Reply:
That sounds good to me